src/app/pages/listing/listing.jsx

```diff
@@ -78,7 +78,7 @@ const useStyles = makeStyles({
   },
   pageHeader: {
     padding: '32px 0',
-    backgroundColor: palette.white,
+    backgroundColor: `${palette.white}`,
```

Comment:
There are 30+ other places in our code where we _don't_ encircle a simple palette.color with the interpolation brackets, including in some other classes in this file. Why did you do it here?

Reply:
I thought this was the standard practice with our palette imports, I will change it back to the previous version.